Program using nested if else

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
+int main() {
+    double n1, n2, n3;
+    printf("Enter three numbers: ");
+    scanf("%lf %lf %lf", &n1, &n2, &n3);
+
+    if (n1 >= n2) {
+        if (n1 >= n3)
+            printf("%.2lf is the largest number.", n1);
+        else
+            printf("%.2lf is the largest number.", n3);
+    } else {
+        if (n2 >= n3)
```

Comment:
it needs to be if else if else ladder then it may work out kindly see that.and make necessary changes